pwiz_tools/Skyline/SkylineResources.resx

```diff
@@ -1163,4 +1163,19 @@
   <data name="CommandLine_AssociateProteins_Associating_peptides_with_proteins" xml:space="preserve">
     <value>Associating peptides with proteins</value>
   </data>
+  <data name="SkylineWindow_ShowImportPeptideSearchDlg_You_must_save_this_document_before_performing_run_peptide_search_" xml:space="preserve">
+	  <value>You must save this document before performing run peptide search.</value>
```

Comment:
This should probably say "before running a peptide search" so that it matches "File > Search > Run Peptide Search".

(the word "performing" would not belong there unless we changed the menu item to "File > Search > Perform Peptide Search")

If you wanted to have both the words "performing" and "run", you would need either some quotes or capitalization or both:
`before performing "Run Peptide Search"
`

Reply:
Done: https://github.com/ProteoWizard/pwiz/pull/3004/commits/bd640eb23c24477debba68b808ebda675d6cfe74